types/user.js

```diff
@@ -0,0 +1,26 @@
+const userType = `
+
+    type Query {
+        users(
+            _id: ID
```

Comment:
I reckon `_id` should be private, and we use generated external UUIDs that never have to change as they will be decoupled from the backend.